collections/_pt_061_enemies/attachment.md

```diff
@@ -6,4 +6,15 @@ level: 100
 start_floor: 65
 end_floor: 67
 patrol: false
+vulnerabilities:
+  bind: 
+  heavy: 
+  sleep: 
+  slow: 
+  stun: true
+abilities:
+  - name: Sewer Water
+    potency: 
+    type: Magic
+    description: 'baited front or rear 180 degree cleave'
```

Comment:
This should be a 180 degree conal followed by a 180 degree rear conal, right? Not one or the other?